hazelcast/src/main/java/com/hazelcast/map/impl/record/AbstractRecord.java

```diff
@@ -106,10 +105,7 @@ public void setHits(long hits) {
     public long getCost() {
         final int objectReferenceInBytes = 4;
```

Comment:
Can you put the 4 into e.g Bits? So that we have a single place where this assumption an address is 4 bytes is stored? If we fix it later; we don't need to go over the whole code base.